vendor/github.com/OpenBazaar/spvwallet/sortsignsend.go

```diff
@@ -629,6 +629,66 @@ func (w *SPVWallet) buildTx(amount int64, addr btc.Address, feeLevel wallet.FeeL
 	return authoredTx.Tx, nil
 }
 
+func NewUnsignedTransaction(outputs []*wire.TxOut, feePerKb btc.Amount, fetchInputs txauthor.InputSource, fetchChange txauthor.ChangeSource) (*txauthor.AuthoredTx, error) {
+
+	var targetAmount btc.Amount
+	for _, txOut := range outputs {
+		targetAmount += btc.Amount(txOut.Value)
+	}
+
+	estimatedSize := EstimateSerializeSize(1, outputs, true, P2PKH)
```

Comment:
Why P2PKH instead of P2SWH? Am I misunderstand how this works?

Reply:
This function only spends from p2pkh atm.